source/includes/_transaction_reporting.md

```diff
@@ -2,6 +2,8 @@
 
 <aside class="notice"><b>IMPORTANT:</b> Note that transaction reporting service is using different domain
 than other ticketswitch endpoints - <b>.ticketswitch.io</b> as opposed to <b>.ticketswitch.com</b></aside>
+<aside class="notice"><b>NOTE:</b>Transaction records are not live data,
+a delay between few seconds and few minutes might be expected from the time when the data has initially entered Ingresso's system</aside>
```

Comment:
"Transaction records are not live data, so a delay of between a few seconds and a few minutes can be expected between the transaction being completed and becoming visible in the reports"

Reply:
cheers, will replace :)

Reply:
updated